src/R/Core/Impl/AST/DataTypes/RString.cs

```diff
@@ -44,5 +44,9 @@ public class RString : RScalar<string> {
         public override int GetHashCode() {
             return this.Value.GetHashCode();
         }
+
+        public override string ToString() {
+            return Value;
```

Comment:
It is partially completed support if we want dropdown-type editing of properties i.e. when Logical gets TRUE/FALSE in the dropdown similar to .NET types. However, I decided not to develop this ability just yet.